src/components/Dropdown/Dropdown.tsx

```diff
@@ -61,9 +67,17 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     screenReaderKey,
     setScreenReaderKey
   );
-  const { focusedIndex, updateFocusedItem } = focusContext;
+  const { focusedIndex, focusedItemData, updateFocusedItem } = focusContext;
 
-  const dropdownContext = useDropdownContextInstance(value, screenReaderUUID, onToggle, onSelect);
+  const dropdownContext = useDropdownContextInstance(
+    lastTypedOrSubmittedValue,
```

Comment:
What is this value when the `Dropdown` is open for the first time and the input is empty?